config/application.rb

```diff
@@ -16,6 +16,9 @@ class Application < Rails::Application
     # Load all of the sub-folders in `app/`
     config.autoloader = :classic
 
+    # Load blacklist of validators from config/validator_blacklist.yml
+    config.validator_blacklist = config_for(:validator_blacklist)
```

Comment:
I think a better place for this would be an initializer and a constant, instead of a config. It's not a config option.